### DISCUSSION TITLE
Why do documentation links keep disappearing?

### DISCUSSION THREAD

**Discussion by ghost** [opendcs/opendcs#846] [Q&A]
I keep documentation links like https://opendcs-env.readthedocs.io/en/stable/lrgs-userguide.html#search-criteria-file-format for my reference.  But a lot of times, these keep disappearing. Why is that? And, more importantly, what is the best way to bookmark these links so they will continue to work?

**Comment by MikeNeilson** [comment]
Odd. This is the first anyone has reported they go missing. I'm not really sure what's going on there and we'll have to look into it. That link should always work. 

That said a few weeks ago we started rearranging the branching and release pattern so it's likely related to that.

**Reply by MikeNeilson** [reply]
Oh, sorry, that one was actually renamed: https://opendcs-env.readthedocs.io/en/stable/legacy-lrgs-userguide.html


We're also trying to overhaul the documentation.
